Source/Example Measurements/NI-DMM Measurement/README.md

```diff
@@ -0,0 +1,21 @@
+# NI-DMM Measurement
```

Comment:
@aepete @bkeryan I stole this from the NI-DMM Python example and removed the information about TestStand sequences. I don't know if they have the same features, so let me know how this might need to change.

Reply:
FYI, I plan on deleting this example. It does not represent best practice. It may get revived at some point. I'm okay with you adding this in case it gets revived, but am also okay with you not adding it.

Reply:
Not adding the README for NI-DMM.